waspc/src/Wasp/CompileOptions.hs

```diff
@@ -22,5 +23,7 @@ data CompileOptions = CompileOptions
     -- CLI commands will almost always compile before they execute to ensure the project is up to date.
     -- This filter function allows callers to ignore certain warnings where they do not make sense.
     -- For example, showing a compilation warning to run `db migrate-dev` when you are running that command.
-    generatorWarningsFilter :: [GeneratorWarning] -> [GeneratorWarning]
+    generatorWarningsFilter :: [GeneratorWarning] -> [GeneratorWarning],
+    -- The user can specify ports when running the command `wasp start`from the cli.
+    startOptions :: StartOptions
```

Comment:
What would be somewhat better is if we would not call these `startOptions`, but `ports` or smth like that. Maybe even individually, each port on its own, not sure though. But it is certainly not important that they are coming from the `start`, we don't want to carry that info all the way into generator, it is CLI detail that rest of Wasp should not need to know about.

Reply:
Good call.  Done.